### PR TITLE
Generic search

### PR DESCRIPTION
**Pull request by xendk**
Can basically take most fields and make it a link to search for the same value. Handy for making the publisher a search for materials from the same publisher, or the same for author, series, genre, or what you can find a Opensearch index for.

### CODE REVIEW

File: ting_material_details.field.inc [hunk 1]
```diff
@@ -464,44 +555,13 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
...
- * Process series information.
- *
- * This could be handled more elegantly if we had better structured data.
- * For now we have to work with what we got to convert titles to links
- * Series information appear in the following formats:
- * - Samhørende: [title 1] ; [title 2] ; [title 3]
- * - [volumne number]. del af: [title]
+ * Quote a string for index search, if it contains anything exotic.
  */
-function process_series_info($series) {
```
Comment (kasperg) [inline ting_material_details.field.inc]: It is intentional that this function has been deleted?

Comment (xendk) [inline ting_material_details.field.inc]: Yeah, it's not used.

**Comment by kasperg** [comment]
Please remember to [create a ticket for the change at platform.dandigbib.dk](http://ting.dk/blog/en-milepael-i-konsolideringen-af-ddb-cms-ding2tal-koden-er-tilbagefoert-til-den-oprindelige-kod) describing the what your proposed change does.

(Yes - we are going to add a `CONTRIBUTING.md` file describing this.)

**Comment by xendk** [comment]
Wouldn't it make more sense if it was the core team that decided which PRs
should be promoted to platform issues? Doesn't make sense to create issues
for needed bugfixes for everybody to look over and approve..

On 13 February 2015 at 12:06, Kasper Garnæs <notifications@github.com>
wrote:

> Please remember to create a ticket for the change at platform.dandigbib.dk
> <http://ting.dk/blog/en-milepael-i-konsolideringen-af-ddb-cms-ding2tal-koden-er-tilbagefoert-til-den-oprindelige-kod>
> describing the what your proposed change does.
>
> (Yes - we are going to add a CONTRIBUTING.md file describing this.)
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ding2/ting_material_details/pull/5#issuecomment-74238063>
> .
>



-- 
Thomas Fini Hansen
fini@reload.dk

**Comment by cableman** [comment]
The issue tracker is not only used to decided what should be part of a given release. It's used to keep track of changes for a given release and used to inform the not so technical persons about bugs and fixes. It should also prevent the same issue being fixed by two different teams etc.

Also it's not the core team as such that decided what goes into a given release, we will advise/recommend changes, but it's the project owners/leaders that makes the final call.

**Comment by xendk** [comment]
Moved to ding2

**Pull request closed without merging**